content/source/docs/enterprise/api/policy-sets.html.md

```diff
@@ -148,16 +223,17 @@ Status  | Response                                      | Reason
 [200][] | [JSON API document][] (`type: "policy-sets"`) | Request was successful
 [404][] | [JSON API error object][]                     | Organization not found, or user unauthorized to perform action
 
-
 ### Query Parameters
 
 This endpoint supports pagination [with standard URL query parameters](./index.html#query-parameters); remember to percent-encode `[` as `%5B` and `]` as `%5D` if your tooling doesn't automatically encode URLs.
 
-Parameter      | Description
----------------|------------
-`page[number]` | **Optional.** If omitted, the endpoint will return the first page.
-`page[size]`   | **Optional.** If omitted, the endpoint will return 20 policy sets per page.
-`search[name]` | **Optional.** Allows searching the organization's policy sets by name.
+Parameter           | Description
+--------------------|------------
+`filter[versioned]` | **Optional.** Allows filtering policy sets based on whether they are versioned (VCS or API), or manually updated. Accepts a boolean true/false value. If omitted, all policy sets are returned.
```

Comment:
```suggestion
`filter[versioned]` | **Optional.** `filter[true]` will only show versioned policy sets (VCS-managed or API-managed), and `filter[false]` will only show manually updated policy sets. If omitted, all policy sets are returned.
```

Reply:
The `versioned` key still needs to be specified so like `filter[versioned]=true` or `filter[versioned]=false`. I hadn't seen this long-form used anywhere else, but I'm not opposed to it.